src/containers/projects/new/step-3.svelte

```diff
@@ -88,7 +88,7 @@
     You are the creator, therefore, also a moderator.
   </p>
   <span class="label">Search for people</span>
-  <Autocomplete values={values} getOptions={getUsers} on:change={recordChanges} />
```

Comment:
I think this should've been replaced for `selection`, because this prop is a way to set the initial selection of values, which will be needed for, say, editing the project.

Reply:
yeah, I didn't really know what it referred to and assumed it was before `getOptions`